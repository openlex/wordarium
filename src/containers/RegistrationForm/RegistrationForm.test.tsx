import React from "react";
import { mount } from "enzyme";
import { RegistrationForm } from "@containers";

describe("test form", () => {
    it("match Snapshot", () => {
        expect(mount(<RegistrationForm />)).toMatchSnapshot();
    });
});
