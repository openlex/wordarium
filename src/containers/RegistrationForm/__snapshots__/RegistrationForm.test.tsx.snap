// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test form match Snapshot 1`] = `ReactWrapper {}`;
